scripts/update_PR_comment.sh

```diff
@@ -26,11 +26,11 @@ if [[ -z $PR ]]; then
   echo "Usage: <new_message> <beginning of message> <pr_number>"
   exit 1
 fi
-data=$(jq -n --arg msg "$message" '{"body": $msg}')
+
 baseURL="https://api.github.com/repos/${GITHUB_REPOSITORY}/issues"
 printf 'Base url: %s\n' "${baseURL}"
 method="POST"
-if [[ -n "$message" ]]; then
+if [[ -f "$message" ]]; then
```

Comment:
I wonder if we should have a "backwards compatibility" `else` clause here that posts the variable content, in case there is no file with the given name.  :shrug:

Reply:
Ah, is the issue that in our sequence of checkouts in `PR_summary.yml`, we're not sure if the version of `update_PR_comment.sh` that runs is from the same commit of mathlib as `PR_summary.yml`? I think I would prefer to address that rather than keep track of backwards compatibility.

Reply:
Honestly, I am not sure what would happen with an old master and the new script: I am already partially surprised that these PRs are running the *new* action/script, so I do not feel too confident in modifying the earlier steps.

Reply:
I think that I would then leave this as is and, if someone has problems with CI, we just tell them to merge master.